apps/web/src/components/Input/Input.tsx

```
@@ -1,3 +1,4 @@
+import { Error } from '@buildeross/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)